src/overlays/actors/ovl_Bg_Jya_Bigmirror/z_bg_jya_bigmirror.h

```diff
@@ -5,6 +5,13 @@
 #include "global.h"
 #include "overlays/actors/ovl_Bg_Jya_Cobra/z_bg_jya_cobra.h"
 
+#define BIGMIR_PUZZLE_COBRA1_SOLVED 1 << 0
+#define BIGMIR_PUZZLE_COBRA2_SOLVED 1 << 1
+#define BIGMIR_PUZZLE_BOMBIWA_DESTROYED 1 << 2
+#define BIGMIR_PUZZLE_IN_STATUE_ROOM 1 << 3
+#define BIGMIR_PUZZLE_IN_1ST_TOP_ROOM 1 << 4
+#define BIGMIR_PUZZLE_IN_2ND_TOP_ROOM 1 << 5
```

Comment:
Ideally, parens should be included in macros like this, to avoid any operator precedence issues
```suggestion
#define BIGMIR_PUZZLE_COBRA1_SOLVED (1 << 0)
#define BIGMIR_PUZZLE_COBRA2_SOLVED (1 << 1)
#define BIGMIR_PUZZLE_BOMBIWA_DESTROYED (1 << 2)
#define BIGMIR_PUZZLE_IN_STATUE_ROOM (1 << 3)
#define BIGMIR_PUZZLE_IN_1ST_TOP_ROOM (1 << 4)
#define BIGMIR_PUZZLE_IN_2ND_TOP_ROOM (1 << 5)
```